README.md

```diff
@@ -21,6 +21,24 @@ If you wish to see visualisations of parse trees, you may optionally install:
 * PyDot https://code.google.com/p/pydot/
 * Pygame https://www.pygame.org/
 
+#### On Ubuntu (including Windows Subsysten for Linux) ####

```

Comment:
Since not every user will come from Ubuntu/WSL, I don't think we need to make this instructions so prevalent. I'm thinking we could either add a Troubleshooting page to the Wiki, or for now just move this to the bottom of the Readme under a `## Troubleshooting` headline. For example:

```
## Troubleshooting

### Installing on Windows Subsystem/Ubuntu

If you have trouble running Eco on the Windows Subsystem, follow these instructions:
```